src/pages/resultsView/survival/SurvivalUtil.tsx

```diff
@@ -77,20 +85,100 @@ export const survivalClinicalDataNullValueSet = new Set([
     'na',
 ]);
 
-export function getEstimates(patientSurvivals: PatientSurvival[]): number[] {
-    let estimates: number[] = [];
+export function getSurvivalSummaries(
```

Comment:
@adamabeshouse could you look at this code especially closely. thanks

Reply:
I don't have a prayer of analyzing this off the top of my head :) @dippindots is there a reference link that you used to write this?

Reply:
Yes, For the standard error calculation, you can see this reference: https://sphweb.bumc.bu.edu/otlt/mph-modules/bs/bs704_survival/BS704_Survival4.html, and for the confidence interval, I am using the `code from R survival package` as the reference: https://github.com/cran/survival/blob/master/R/survfit.R#L308-L316